### ISSUE TITLE
Release enhanced-table-1.9.0_7.6.0.zip dependency issues?

### ISSUE THREAD

**Issue by fonfalleh** [fbaligand/kibana-enhanced-table#115]
```
Jul 10 11:34:22 computar kibana[58369]:     ERROR in ./plugins/enhanced-table/public/document-table-vis-data-params.js
Jul 10 11:34:22 computar kibana[58369]:     Module not found: Error: Can't resolve 'ui-select' in '/blah/kibana/plugins/enhanced-table/public'
```
I know next to nothing about js, but the extracted enhanced-table-1.9.0_7.6.0.zip has no reference to `ui-select` in package.json, as opposed to the 7.6 branch; perhaps that's it?

**Comment by fbaligand** [comment]
Hi,

I just tried to install enhanced-table-1.9.0_7.6.0.zip on Kibana 7.6.0, and it works great.
ui-select is a dependency provided by kibana core in $KIBANA_HOME/node_modules/ui-select.
So there is normally no need to add it into plugin itself

**Comment by fbaligand** [comment]
Hi,

Even I don't reproduce the issue, I just released kibana v1.9.1, where packages for kibana 7.6 now have ui-select dependency embedded.
